mavlink_interface compile warning, possible bug?

Hello everyone!

When I'm compiling the mavlink_interface object, I get an error for this line:
https://github.com/PX4/PX4-SITL_gazebo/blob/06e801fe8b5267e561f00d8847a6343cb2386ab6/src/mavlink_interface.cpp#L433-L439

The error message is:
```
<project_dir>/src/mavlink_interface.cpp: In member function ‘void MavlinkInterface::send_mavlink_message(const mavlink_message_t*)’:
<project_dir>/src/mavlink_interface.cpp:513:21: warning: comparison of unsigned expression < 0 is always false [-Wtype-limits]
  439 |             if (len < 0)
      |                 ~~~~^~~
```
Should < be == perhaps? What is the error result of the preceding `send` function?

--- a/src/mavlink_interface.cpp
+++ b/src/mavlink_interface.cpp
@@ -430,7 +430,7 @@ void MavlinkInterface::send_mavlink_message(const mavlink_message_t *message)
         return;
       }
 
-      size_t len;
+      ssize_t len;
       if (use_tcp_) {
         len = send(fds_[CONNECTION_FD].fd, buffer, packetlen, 0);
       } else {

